examples/templates/exposed-thing/src/base.ts

```diff
@@ -127,13 +145,14 @@ export class WotDevice {
     	*/
     }
 
-    private add_properties() {
+    private addProperties() {
         //fill in add properties
-        this.thing.writeProperty("myProperty", ""); //replace quotes with the initial value
-        this.thing.setPropertyReadHandler("myProperty", this.myPropertyHandler);
+        this.myProperty = ""  ; // replace quotes with the initial value
+        this.thing.setPropertyReadHandler("myProperty", this.myPropertyReadHandler);   // not applicable for write-only
+        this.thing.setPropertyWriteHandler("myProperty", this.myPropertyWriteHandler); // not applicable for read-only
     }
 
-    private add_actions() {
+    private addActions() {
```

Comment:
contrary to my comment about properties, this should be called `addActionHandlers`